### PR TITLE
feat: separate dome lights

### PR DESCRIPTION
**Pull request by tyler58546**
## Summary of Changes

* Separated dome light into two separate objects that can be independently controlled in the future.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Make sure the dome light still works as expected.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by davidwalschots — APPROVED** [review]
Very nice!

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 16/4/21
Version of the sim: 1.15.8.0
PR Tested: #4462 
Tier of testing: 1
Changes to observe:
Dome Light

Testing technique: Loaded in at different stages and tried
Steps you've made to test the Pull Request:
1. Loaded in C&D, started up and used dome lights
2. Loaded on runway, repeated^
3. Loaded in the air, same^
4. Loaded on approach, and again^

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here.
image
image

Issues (If Any): When loading up mid-air the position of the switch is "Bright" but the dome light is off, when the switch is moved to "Dim" the light starts working correctly again
Steps to achieve the issue: Load in the air and look at the dome light switch position and the dome light

Overall Rating: To be improved
Conclusions of the testing: There were no instructions so I guessed I had to look if it was still working, works correctly but bugs out when loading mid-air.

**Comment by tyler58546** [comment]
> Issues (If Any): When loading up mid-air the position of the switch is "Bright" but the dome light is off, when the switch is moved to "Dim" the light starts working correctly again
> Steps to achieve the issue: Load in the air and look at the dome light switch position and the dome light

I just tested it and the issue is not caused by this PR.

File: src/model/a320-interior/A320_NEO_INTERIOR_LOD00.gltf [hunk 1]
```diff
@@ -187213,7 +187213,6 @@
                 66,
                 67,
                 70,
-                73,
```
Comment (pepperoni505) [inline src/model/a320-interior/A320_NEO_INTERIOR_LOD00.gltf:187216]: Shouldn't you remove it on all LODs?

Comment (tyler58546) [reply src/model/a320-interior/A320_NEO_INTERIOR_LOD00.gltf]: I decided to just hide it using modelbehaviors so I don't have to modify any original model files.

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 18.04.2021
Version of the sim: 1.15.8.0
PR Tested: #4462
Tier of testing: 1
Changes to observe: dome light function

Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned C&D at EDDP
2.) Turned on bat. and ext. power
3.) Test dome light
4.) Test other light switches will turn on dome light
5.) Spawned on runway at EDDP
6.) Test dome light

Issues: -

Overall Rating: Good

Conclusions of the testing: Works

Hope this helps :-)

**Pull request merged**